db/app.rb

```diff
@@ -0,0 +1,25 @@
+require 'sinatra'
+require 'sinatra/reloader'
+
+require_relative 'db_connection'
+require_relative 'models/apartment'
+require_relative 'models/tenant'
+require_relative 'controllers/welcome'
+require_relative 'controllers/apartments'
+require_relative 'controllers/tenants'
+
+# get '/' do
```

Comment:
the routes would work for those apartment views, you just need to query the database for objects.